tests/test-e2e-apps/bridge-server/opampsrv/opampsrv.go

```diff
@@ -60,7 +60,7 @@ func (srv *Server) Start() {
 				},
 			},
 		},
-		ListenEndpoint: "127.0.0.1:4320",
+		ListenEndpoint: "0.0.0.0:4320",
```

Comment:
this was causing a failure to connect to the endpoint... i may need to split this out in order for the server to work